File: recyclerviewespresso/src/main/java/com/allodel/recyclerviewespresso/MainActivity.java
package com.allodel.recyclerviewespresso;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

public class MainActivity extends AppCompatActivity {

    private RecyclerView contactRecyclerView;
    private ContactAdapter contactAdapter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        contactRecyclerView = findViewById(R.id.rv_contacts);

        contactAdapter = new ContactAdapter(new ContactPozo().getContacts());
        contactRecyclerView.setLayoutManager(new LinearLayoutManager(this));
        contactRecyclerView.setAdapter(contactAdapter);
    }
}

File: testingcomplexui/src/main/java/com/allodel/testingcomplexui/MainActivity.kt
package com.allodel.testingcomplexui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        setTitle(resources.getString(R.string.app_name))
    }
}
